api/testrun.go

```diff
@@ -135,6 +135,9 @@ func (c *Client) TestRunLogs(path string, preview bool) (io.ReadCloser, error) {
 
 	if response.StatusCode != 200 {
 		response.Body.Close()
+		if response.StatusCode == 404 {
+			return nil, errors.New("test-case or logs not found")
```

Comment:
That should be test run not found, right?

Reply:
Done